test/composertest.py

```diff
@@ -64,12 +53,13 @@ def setUp(self):
                                 "--silent",
                                 "--unix-socket", "/run/weldr/api.socket",
                                 "http://localhost/api/status"]
-        r = subprocess.call(self.ssh_command + curl_command, stdout=open(os.devnull, 'w'))
+        r = subprocess.run(self.ssh_command + curl_command, stdout=subprocess.DEVNULL)  # pylint: disable=no-member
         self.assertEqual(r.returncode, 0)
 
     def tearDown(self):
-        # `errors` is a list of tuples (method, error)
-        errors = list(e[1] for e in self.ci_result.errors if e[1])
+        # Peek into internal data structure, because there's no way to get the
+        # TestResult at this point. `errors` is a list of tuples (method, error)
+        errors = list(e[1] for e in self._outcome.errors if e[1]) # pylint: disable=no-member
```

Comment:
We're running this under python3. There's no need for the `no-member` pylint rule.

Reply:
There's pylint running under Python2 (to inspect the source code) which also discovers these files and complains about them. Hence the no-member disable.

Reply:
Please don't resolve conversations as the one who is supplying information. The person who asked a question is the one to do that, when the answer does in fact resolve the conversation. (Also, GitHub hides the answer in the UI, which makes it hard to see.)

I don't believe I have to say this, but you cannot check python 3 code with a python 2 linter.

Reply:
Actually you can because the linter doesn't know what is the language version you intended these source files to be run with. When resolving the attributes it tries to load the respective module/class and when it doesn't find the desired attributes it complains.

I see a few options here: 

1) Configure the linter to ignore these files or
2) Don't copy them inside the docker image in the first place or
3) Run a linter with Python 3 to also examine these scripts ?

Also about the already changed test modules:

1) Revert everything to be exactly the same with master or
2) Remove the pylint: disable comments and leave the code as-is, which includes some differences that I've made initially

Tell me which one you want to have and I will update the code.

Reply:
We should just make it skip the test scripts.

Reply:
I've reverted the changes made to satisfy the linter essentially making these files looks the same as on master branch. I've added an `ignoreNames` attribute to the linter configuration. Will adjust the format if Travis CI reports failure.